test/helpers/cli-server.js

```diff
@@ -32,12 +32,12 @@ Server.prototype.start = function() {
     });
 
     server.on('exit', function(statusCode, signal) {
-      if (signal === 'SIGTERM') {
+      if (signal === 'SIGTERM' || statusCode === 143) {
```

Comment:
@rwjblue This change is just because on Node 0.10 it returns a status code of 143 instead of the string `SIGTERM`.